(GH-33) Draft rules for Taunts (#37)
FlagrantGarden/site: content/games/factions/rules.md

|  Fortify   |  7+   |  18”  |    Another Group    | Until the start of your next turn. | Target may force enemies to reroll any number of to-hit dice once against them. |
|  Counter   |  8+   |  18”  |        Group        |                 -                  | End any Spell effect on the target as if its duration ended.                    |

#### Taunts

| Taunt                          | Check | Requirements                                     | Effect                                                                                                                                 |
| :----------------------------- | :---: | :----------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------- |

> | Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS if they would otherwise do so) and all enemy Groups are Reckless against them.                           |

Reviewer: ```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so), and all enemy Groups are Reckless against them.                           |
```

Author: Should that comma be there or should it read:
> Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.
```suggestion
| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS instead if they would be forced to do so) and all enemy Groups are Reckless against them.                           |
```

Reviewer: either is fine actually. now im looking at it, the ethos of this doc probably leans toward your suggestion.

| We Are Unbreakable!            |  6+   | -                                                | Roll 3d6 and take the highest when testing Resolve but on failure, immediately Rout.                                                   |
| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group              | While within 6" of a friendly Group re-roll any dice but lose 1d6 FS whenever a friendly Group within 6" Routs                         |

> | We Never Miss a Shot!          |  6+   | Groups with an MI profile only.                  | Must Shoot at the furthest enemy Group in range (if any) but may re-roll any misses when Shooting.                                     |

Reviewer: ```suggestion
| We Never Miss a Shot!          |  6+   | Groups with an MI profile only.                  | When this Group activates to Shoot, they must Shoot the farthest enemy Group in range (if any) but may reroll any misses when Shooting.                                     |
```

> | We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |

Reviewer: ```suggestion
| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group.               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |
```

> | We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |

Reviewer: ```suggestion
| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group.               | While within 6" of a friendly Group, this Group may reroll any dice. This Group loses 1d6 FS whenever a friendly Group within 6" Routs.                                                           |
```

> | We'll Never Taste Your Blades! |  6+   | -                                                | Gain +1 T and +1 on to-hit rolls when Attacking but always count hits against them twice.                                              |

Reviewer: ```suggestion
| We'll Never Taste Your Blades! |  6+   | -                                                | +1 T and +1 on to-hit rolls when Attacking, but hits against this Group always count twice.                                              |
```

> | You'll Die with Us!            |  6+   | Must be Bloodied                                 | Inflict one hit for each taken when being Attacked but -1 T.                                                                           |

Reviewer: ```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied                                 | When Attacked, this Group inflicts one hit against their attackers for each hit they take, but have -1 T.                                                                           |
```

> | We'll Never Taste Your Blades! |  6+   | -                                                 | Gain +1 T and +1 on to-hit rolls when Attacking. Hits against this Group always count twice.                                                                                                      |

Reviewer: ```suggestion
| We'll Never Taste Your Blades! |  6+   | -                                                 | When attacking, this Group has +1 T and +1 on to-hit rolls. Hits against this Group always count twice.                                                                                                      |
```

> | There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group; when Attacking them gain +1 T and treat them as half as far away but must activate to Attack or Move toward them. |

Reviewer: ```suggestion
| There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T, and treat this Group's MO as double its written value. |
```

Author: Is the second comma needed here, or should it be:
> When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value.
I _think_ if we removed the conditional at the start we wouldn't have that second comma, right?
```suggestion
| There Can Be Only One! | 7+ | - | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value. |
```

> | We'll Fight in the Shade!      |  7+   | -                                                | Gain +1 T when an enemy Shoots them but gain Reckless trait against enemy Groups with an MI profile.                                   |

Reviewer: ```suggestion
| We'll Fight in the Shade!      |  7+   | -                                                | This Group has +1 T when an enemy Shoots them, but gain the Reckless trait against enemy Groups with an MI profile.                                   |
```

> | You'll Die with Us!            |  6+   | Must be Bloodied                                  | Lose -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                             |

Reviewer: ```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied                                  | -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                             |
```

> | You'll Die with Us!            |  6+   | Must be Bloodied                                  | -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                                  |

Reviewer: ```suggestion
| You'll Die with Us!            |  6+   | Must be Bloodied.                                  | -1 T. When Attacked, this Group inflicts one hit against their attackers for each hit they take.                                                                                                  |
```

> | You'll Never Take Us Alive!    |  7+   | -                                                | Cannot Retreat (-1 FS if they would otherwise do so) but +1 on to-hit rolls in when Attacking or being Attacked.                       |

Reviewer: ```suggestion
| You'll Never Take Us Alive!    |  7+   | -                                                | Cannot Retreat (-1 FS instead if they would be forced to do so), but +1 on to-hit rolls when Attacking or being Attacked.                       |
```

> | For the Captain!               |  8+   | Must be within 12" of Captain                    | While within 12" of Captain re-roll all failed Activations; otherwise re-roll all successful Activations and Resolve tests.            |

Reviewer: ```suggestion
| For the Captain!               |  8+   | Must be within 12" of Captain.                    | While within 12" of Captain, this Group rerolls all failed Activations. Otherwise, reroll all successful activations and Resolve tests.            |
```

> | We'll Run You Down!            |  8+   | Groups with an MO of at least 8" only.           | Gain the Reckless trait and increase their MO by 2" but may only activate to Attack or Move.                                           |

Reviewer: ```suggestion
| We'll Run You Down!            |  8+   | Groups with an MO of 8" or more only.           | This Group gains the Reckless trait, +2" MO, and may only activate to Attack or Move.                                           |
```

> | You'll Never Take Us Alive!    |  7+   | -                                                 | This Group cannot Retreat (-1 FS instead if they would be forced to do so) and gains +1 on to-hit rolls when Attacking or being Attacked.                                                         |

Reviewer: ```suggestion
| You'll Never Take Us Alive!    |  7+   | -                                                 | This Group cannot Retreat (-1 FS instead if they would be forced to do so). They have +1 on to-hit rolls when Attacking or being Attacked.                                                         |
```

> | Your Magic is Laughable!       |  8+   | Groups with the Apprentice or Caster trait only. | May choose to force an enemy to reroll a Cast activation; any spell which still succeeds is twice as effective.                        |

Reviewer: ```suggestion
| Your Magic is Laughable!       |  8+   | Groups with the Apprentice or Caster traits only. | This Group may force an enemy to reroll a Cast activation against them. Any spell which still succeeds is twice as effective.                        |
```

Reviewer: Heads up: this is a functional change ("against them" to limit use a bit). It seemed to need some direction, could have been "a Cast activation within 12"" or something instead. I think I like this one better.

   Shaken Groups only hit on a 6.
   For each number of hits equal to the enemy’s T, reduce the enemy’s FS by 1.

#### Proclaim

> Any Group may choose to Proclaim a taunt, introducing a new special rule affecting them for the remainder of the skirmish.

Reviewer: ```suggestion
A Group may activate to Proclaim a taunt, potentially introducing a new special rule affecting them for the remainder of the skirmish.
```

> To activate, they must roll at least the Check value for the taunt they wish to Proclaim.

Reviewer: ```suggestion
To Proclaim, the Group must activate with roll that meets or exceeds the Check value for the specific Taunt they wish to Proclaim.
```
